config/s-nt.h

```diff
@@ -17,7 +17,7 @@
 
 #define OCAML_OS_TYPE "Win32"
 
-#ifdef __MINGW32__
+#if defined(__MINGW32__) || _MSC_VER >= 1200
```

Comment:
This **should** be `1600`.

`stdint.h` is only available from VS 2010 - the `1200` was to do with the `_UINTPTR_T_DEFINED` guards (since for `_MSC_VER < 1200`, `stddef.h` doesn't include the `intptr_t` types)

Reply:
Ok, thanks ! It seems we are converging.